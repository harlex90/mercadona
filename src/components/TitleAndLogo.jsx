import caddie from "../assets/caddie.jpg";

const entete = {
    title: "MERCADONA",
    caption: "Des promos toute l'annÃ©e !",
    image: caddie,
};

const TitleAndLogo = () => {
    return(
        <div style={{display: "flex", flex: "1 1 10%", justifyContent: "center", gap: 10, alignItems: "center"}}>
            <div style={{display: "flex"}}>
                <img
                    src={entete.image}
                    alt={"caddie"}
                    style={{display: "flex", width: 50, height: 50, borderRadius: "50%", border: '2px solid black'}}
                />
            </div>
            <div style={{display: "block"}}>
                <h1 style={{margin: 0}}>
                    {entete.title}
                </h1>
                <h4 style={{fontStyle: "italic", margin: 0}}>
                    {entete.caption}
                </h4>
            </div>
        </div>
    )
}

export default TitleAndLogo;

