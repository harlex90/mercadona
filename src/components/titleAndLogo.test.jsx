import { render } from '@testing-library/react';
import TitleAndLogo from './TitleAndLogo';

describe('TitleAndLogo', () => {
    test('match snapshot', () => {
        const { asFragment } = render(<TitleAndLogo />);
        expect(asFragment()).toMatchSnapshot()
    });
    test('display title and logo', () => {
        const { getByText } = render(<TitleAndLogo />);
        expect(getByText("MERCADONA")).toBeInTheDocument();
        expect(getByText("Des promos toute l'ann√©e !")).toBeInTheDocument();
    });
});