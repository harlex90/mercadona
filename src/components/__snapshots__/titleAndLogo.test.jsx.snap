// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleAndLogo match snapshot 1`] = `
<DocumentFragment>
  <div
    style="display: flex; flex: 1 1 10%; justify-content: center; gap: 10px; align-items: center;"
  >
    <div
      style="display: flex;"
    >
      <img
        alt="caddie"
        src="caddie.jpg"
        style="display: flex; width: 50px; height: 50px; border-radius: 50%; border: 2px solid black;"
      />
    </div>
    <div
      style="display: block;"
    >
      <h1
        style="margin: 0px;"
      >
        MERCADONA
      </h1>
      <h4
        style="font-style: italic; margin: 0px;"
      >
        Des promos toute l'ann√©e !
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
